### PR TITLE
image-input done

### PR DESCRIPTION
**Pull request by kushalsourav**
image and input component finished

### CODE REVIEW

**Comment by BALAJI141031** [comment]
inputs are good to go

File: packages/components/image.css [hunk 1]
```diff
@@ -0,0 +1,8 @@
+.img-responsive {
+    width: 100%;
+    height: 100%;
```
Comment (deekshithmd) [inline packages/components/image.css:3]: ig, for height it will be auto. check once

File: packages/components/input.css [hunk 2]
```diff
@@ -0,0 +1,77 @@
+.input-label {
+    display: inline-block;
+    text-transform: capitalize;
+    font-weight: 400;
+    margin: 0.625rem 0;
```
Comment (deekshithmd) [inline packages/components/input.css:5]: using px will be good

**Review by deekshithmd — APPROVED** [review]

File: packages/components/image.css (hunk 1 above)
Comment (ashishkatiyar2404) [reply packages/components/image.css:3]: you are right deekshithmd.

**Review by ashishkatiyar2404 — CHANGES_REQUESTED** [review]

**Review by ashishkatiyar2404 — CHANGES_REQUESTED** [review]
1. add cursor: pointer to the submit button of Form Validation
2. your responsive img doesn't look to be responsive(its width decrease but height still reamain the same)
3. add side nav bar or make ur navbar sticky 
4. on hovering your coponent button (again here add cursor pointer)

**Review by ashishkatiyar2404 — APPROVED** [review]
other all look good.
clean UI.

**Pull request merged**